x-pack/plugin/ml/qa/single-node-tests/src/test/java/org/elasticsearch/xpack/ml/transforms/PainlessDomainSplitIT.java

```diff
@@ -266,7 +267,7 @@ public void testHRDSplit() throws Exception {
                 "\"time\": { \"type\": \"date\" } } }");
 
         // Index some data
-        DateTime baseTime = new DateTime().minusYears(1);
+        ZonedDateTime baseTime = ZonedDateTime.now().minusYears(1);
```

Comment:
`ZonedDateTime.now(ZoneOffset.UTC)` should be used, as IIRC the system timezone will be used otherwise

Reply:
you are right, the system default timezone will be used. But woudn't this be change of behaviour if we use the UTC here? it was using new DateTime() previosly - meaning system default time zone was used as well

Reply:
For this particular test the timezone shouldn't matter provided the text representation of the timestamps includes the timezone and the test doesn't cross the start/end of daylight saving time.  (I think the bit about not spanning daylight saving is a bug in the current version of the test.)

So I don't think it would be a problem to always do the test in UTC.  This is not primarily testing date/time functionality so there's no need to get coverage of different timezones.  It's testing completely different functionality but just needs to generate incrementing timestamps because anomaly detection deals with time series data.

Reply:
thanks a ton for the clarification Dave, much appreciated!

I'd like to add all methods not specifying a timezone to the forbidden apis over time, thus my request.